grpc-proto/proto/schema.proto

```diff
@@ -32,11 +32,23 @@ enum ColumnOrderBy {
 
 // INDEXES
 
+enum IndexingType {
+  DEFAULT = 0;
```

Comment:
The indexing type is optional. I opted for a default "empty" value rather than adding a wrapper message (not even sure if protobuf provides an equivalent of `StringValue` for enums out of the box).

Reply:
+1 for explicit value, given that we don't have to worry about backwards compatibility yet.